platforms/chibios/eeprom_stm32_defs.h

```diff
@@ -72,3 +72,75 @@
 #        define FEE_PAGE_BASE_ADDRESS ((uintptr_t)(FEE_FLASH_BASE) + FEE_MCU_FLASH_SIZE * 1024 - (FEE_PAGE_COUNT * FEE_PAGE_SIZE))
 #    endif
 #endif
+
+/* These bits are used for optimizing encoding of bytes, 0 and 1 */
+#define FEE_WORD_ENCODING 0x8000
+#define FEE_VALUE_NEXT 0x6000
+#define FEE_VALUE_RESERVED 0x4000
+#define FEE_VALUE_ENCODED 0x2000
+#define FEE_BYTE_RANGE 0x80
```

Comment:
These defines are not used in the size calculations below and may stay internal (although their values are related to `FEE_ADDRESS_MAX_SIZE`, because that value is determined by the encoding used here, and `FEE_ADDRESS_MAX_SIZE` is needed in the size checks).